components/UserProvider.js

```diff
@@ -98,10 +98,10 @@ class UserProvider extends React.Component {
    * [refetchQueries](https://www.apollographql.com/docs/react/api/react-apollo.html#graphql-mutation-options-refetchQueries)
    * if you really need to be up-to-date with server.
    */
-  refetchLoggedInUser = async () => {
+  refetchLoggedInUser = async ({ ignoreApolloCache = false } = {}) => {
```

Comment:
I'd prefer having `ignoreApolloCache` to `true` by default: the expectation I have when calling `refetchLoggedInUser` is to to make a server call. I'll probably forget to pass `ignoreApolloCache` next time I have to call it.

I'm not sure why we would call this function to refetch from the cache, but if there's an usage for it I don't think that it should be the default.

Reply:
Agree, crazy that as of today `refetchLoggedInUser` is not hitting the network!

I'll update the PR.